.github/workflows/weekly-canary-build.yml

```diff
@@ -0,0 +1,44 @@
+name: Weekly Canary Build
+
+on:
+    schedule:
+        - cron: '0 0 * * Mon'
+
+jobs:
+  weekly-canary-build:
+    strategy:
+        matrix:
+            rust-channel: [stable, beta, nightly]
+    runs-on: ubuntu-20.04
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install tools
+        uses: taiki-e/install-action@v2
+        with:
+          tool: mdslides@0.3,mdbook@0.4,mdbook-mermaid@0.12,flip-link@0.1.7
```

Comment:
We should update both repos to use the latest flip-link, which is 0.1.10.

Reply:
Send one PR for [rust-exercises](https://github.com/ferrous-systems/rust-exercises/pull/163) but I don't see any mention of `flip-link` in the rust-training repo.

Did you mean 'both actions" perhaps?